lib/config.js

```diff
@@ -0,0 +1,11 @@
+/*!
+ * Copyright (c) 2019 Digital Bazaar, Inc. All rights reserved.
+ */
+'use strict';
+
+const {config} = require('bedrock');
+
+const cfg = config['jsonld-document-loader'] = {};
+
+// contains a list of constants and their associated canonical URL
+cfg.urls = {};
```

Comment:
Is this what we want ... or did we want to just keep using `bedrock.constants` for this?

Reply:
config removed, we will make no change to how contexts URLs are handled.